relay-dynamic-config/src/global.rs

```diff
@@ -7,16 +7,25 @@ use crate::TaggingRule;
 ///
 /// Values shared across all projects may also be included here, to keep
 /// [`ProjectConfig`](crate::ProjectConfig)s small.
-#[derive(Default, Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize)]
 #[serde(default, rename_all = "camelCase")]
 pub struct GlobalConfig {
     /// Project configuration for measurements.
     #[serde(skip_serializing_if = "Option::is_none")]
-    measurements: Option<MeasurementsConfig>,
+    pub measurements: Option<MeasurementsConfig>,
     /// Project configuration for rules for applying metrics tags depending on
     /// the event's content.
-    #[serde(skip_serializing_if = "Option::is_none")]
-    metric_conditional_tagging: Option<Vec<TaggingRule>>,
+    #[serde(skip_serializing_if = "Vec::is_empty")]
+    pub metric_conditional_tagging: Vec<TaggingRule>,
+}
+
+impl Default for GlobalConfig {
+    fn default() -> Self {
+        Self {
+            measurements: Some(MeasurementsConfig::default()),
+            metric_conditional_tagging: Default::default(),
+        }
+    }
```

Comment:
added this just for the integration test, otherwise just an empty dict is returned since we don't have a proper default yet, which makes it hard to know if the integration test is succesful